packages/host/app/commands/request-forward.ts

```diff
@@ -0,0 +1,69 @@
+import { service } from '@ember/service';
+
+import type * as BaseCommandModule from 'https://cardstack.com/base/command';
+
+import HostBaseCommand from '../lib/host-base-command';
+
+import type RealmServerService from '../services/realm-server';
+
+export default class RequestForwardCommand extends HostBaseCommand<
```

Comment:
```suggestion
export default class SendRequestViaProxyCommand extends HostBaseCommand<
```

I think this name is better, and we should update corresponding input, result, etc